### ISSUE TITLE
Ads collected on Chrome, not on Firefox

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdNauseam#765]
![screen shot 2017-01-15 at 12 24 09 pm](https://cloud.githubusercontent.com/assets/737638/21960225/bb2bce82-db1d-11e6-8b3b-15500487107b.png)

![screen shot 2017-01-15 at 12 23 54 pm](https://cloud.githubusercontent.com/assets/737638/21960227/c66ad810-db1d-11e6-801c-45b0ddf8f06a.png)

**Comment by cqx931** [comment]
This might not be a valid firefox problem. 
There is a large chance that ads shown on this site are dynamically animated and those ads could not be collected by adnauseam.
![image](https://cloud.githubusercontent.com/assets/3056154/22008843/dddf177a-dcb9-11e6-97bd-a55844227093.png)
If the ad is a proper image ad, firefox is able to collect it properly.

**Comment by dhowe** [comment]
what do you mean by "dynamically animated" ? you mean a text-ad that is updated over time ?

**Comment by cqx931** [comment]
In the above two ads, the first one is an "dynamically animated" one.
There are images and text and the layout and animation is done with css and javascript.
The second one is a text-ad.

**Comment by dhowe** [comment]
Am I correct then that these ads are not collected on Chrome OR Firefox?

**Comment by cqx931** [comment]
yes,they are not collected in any browser. It is kind of tricky to decide what shall be collected here... a similar case is ad drawn in a canvas.

**Comment by dhowe** [comment]
We're not seeing enough of this style of ad (either kind) to make it worth the time, however, it is a Google-served ad so it may be a sign of the future -- in any case, i will move this to possible futures and we can keep an eye open for similar cases...

**Comment by cqx931** [comment]
I think the trend is these two kinds of ad are becoming more and more popular, as it is faster to load than images. Canvas might be easier to deal with if we could just save one frame from the canvas. But I am not sure whether we could do something like a  'screen shot' for specific dom element.

**Comment by dhowe** [comment]
right, i've thought about this for videos as well... but lets keep on hold for now

**Comment by cqx931** [comment]
More test cases: (the content inside the iframe)
http://lab-lamp.scm.cityu.edu.hk/adntest/dynamicCases/case1.html
http://lab-lamp.scm.cityu.edu.hk/adntest/dynamicCases/case2.html
